Source/Dafny/Verifier/Translator.BoogieFactory.cs

```diff
@@ -0,0 +1,872 @@
+
```

Comment:
Consider naming this file `Translator.BoogieUtils.cs` or something like that. `BoogieFactory` made me expect to see a class with that name, whereas this is just a slice of utility methods in the partial `Translator` class.

Reply:
The methods are only for creating Boogie programs, that's why I called it `Factory` which is a bit more specific than `Utils`. Wouldn't `BoogieUtils` make you expect to see a class with that name?